benchmark/deepseek_v3/README.md

```diff
@@ -24,7 +24,7 @@ docker run --gpus all --shm-size 32g -p 30000:30000 -v ~/.cache/huggingface:/roo
 ### Using pip
 ```bash
 # Installation
-pip install "sglang[all]" --find-links https://flashinfer.ai/whl/cu124/torch2.4/flashinfer
+pip install "sglang[all]==0.4.1.post1" --find-links https://flashinfer.ai/whl/cu124/torch2.4/flashinfer
```

Comment:
We should update the version here. Otherwise, if the user has already installed it and uses the original command again, the local SGLang won't be updated. cc @ispobock @HandH1998